.github/workflows/ci.yml

```
@@ -167,7 +167,11 @@ jobs:
         source .venv/bin/activate
         mkdir "$NLTK_DATA"
         make install-ci
-    - name: Test
+    - name: Test (unit)
```

```suggestion
    - name: Test Ingest (unit)
```
(just to make it clearer in github actions UI)